core/src/main/java/io/keen/client/java/ScopedKeys.java

```diff
@@ -63,7 +63,11 @@
      */
     public static String encrypt(String apiKey, Map<String, Object> options)
             throws ScopedKeyException {
-        return encrypt(KeenClient.client(), apiKey, options);
+        if (apiKey.length() == 64) {
+            return encrypt(KeenClient.client(), apiKey, options);
+        } else {
+            return oldEncrypt(KeenClient.client(), apiKey, options);
```

Comment:
Suggestion: maybe call these `encrypt64CharacterKey` and `encrypt32CharacterKey` or something instead of `encrypt` and `oldEncrypt`; the latter doesn't provide a lot of insight into the difference, and will just need to get renamed again if/when we change the algorithm again.